xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Recordings.h

```diff
@@ -492,6 +496,35 @@ class PVRRecording : public CStructHdl<PVRRecording, PVR_RECORDING>
   /// @brief To get with @ref SetProviderName changed values.
   std::string GetProviderName() const { return m_cStructure->strProviderName; }
 
+  /// @brief **optional**\n
+  /// Age rating for the recording.
+  void SetParentalRating(int iParentalRating) { m_cStructure->iParentalRating = iParentalRating; }
+
+  /// @brief To get with @ref SetParentalRating changed values
+  int GetParentalRating() const { return m_cStructure->iParentalRating; }
+
+  /// @brief **optional**\n
+  /// Parental rating code for this recording.
+  void SetParentalRatingCode(const std::string& ratingCode)
+  {
+    strncpy(m_cStructure->strParentalRatingCode, ratingCode.c_str(),
+            sizeof(m_cStructure->strParentalRatingCode) - 1);
+  }
+
+  /// @brief To get with @ref SetParentalRatingCode changed values.
+  std::string GetParentalRatingCode() const { return m_cStructure->strParentalRatingCode; }
```

Comment:
Commit message only talks about introduction of parental rating icon. Actually this commit contains much more changes, like  introduction of more functionality for recordings. The latter should be a separate commit of this PR.